RHDEVDOCS-4719+4724 Improve Tekton Hub config UX + Support for Crunchy Postgres with Tekton Hub (#54522)
openshift/openshift-docs: modules/op-adding-new-users-in-tekton-hub-configuration.adoc

:_content-type: PROCEDURE
[id="adding-new-users-in-tekton-hub-configuration_{context}"]
= Adding new users in {tekton-hub} configuration

[role="_abstract"]

> Cluster administrators with `config:refresh` scope can add new users to {tekton-hub} with different scopes.

Reviewer: ```suggestion
Cluster administrators can add new users to {tekton-hub} with different scopes.
```

[discrete]
.Procedure

> . Modify the `TektonConfig` CR to add new users with different scopes.

Reviewer: ```suggestion
. Modify the `TektonHub` CR to add new users with different scopes.
```

+
[NOTE]
====

> A new user signing in to {tekton-hub} for the first time will have only the default scope. To activate additional scopes, ensure the user has logged in at least once.

Reviewer: ```suggestion
A new user signing in to {tekton-hub} for the first time will have only the default scope. To activate additional scopes, ensure the user's username is added in the scopes i.e. in the TektonHub CR
```
